ext/data/schemas/dictionary-index-schema.json

```diff
@@ -21,7 +21,7 @@
         },
         "revision": {
             "type": "string",
-            "description": "Revision of the dictionary. This value is only used for displaying information."
+            "description": "Revision of the dictionary. This value is displayed, and used to check for dictionary updates."
```

Comment:
What do you think about using an `updatedDate` field? I feel like it could be more robust than using the revision for comparison.

Reply:
Wouldn't mind that either, but think it's redundant as we already have a field for versioning and not everyone might want to use calver.

Reply:
I'm thinking about the (uncommon) scenario where you might want to change your versioning scheme for whatever reason. Like, if you have a really long string `dictionary_info_version_2024-07-12` and want to update that to `2024-07-13`, which you can't. Or if your previous versioning scheme was just lacking in any way.

To be fair, the ones I have bad versioning with are all just `1.0.0` which is considered less than `2024-07-12` by JS so it's probably fine.

Reply:
Hmm yeah, I guess changes like that will have to be made in the first revision that "opts in" to being updatable. The dict-creating docs will recommend using dot-separated numbers for revision field for new dicts (so either calver or semver)